qiskit/visualization/state_visualization.py

```diff
@@ -807,20 +824,20 @@ def plot_state_qsphere(rho, figsize=None, ax=None):
     ax2 = fig.add_subplot(gs[2:, 2:])
     ax2.pie(theta, colors=sns.color_palette("hls", n), radius=0.75)
     ax2.add_artist(Circle((0, 0), 0.5, color='white', zorder=1))
-    ax2.text(0, 0, 'Phase', horizontalalignment='center',
+    ax2.text(0, 0, 'Phase\n(Deg)', horizontalalignment='center',
              verticalalignment='center', fontsize=14)
 
     offset = 0.95  # since radius of sphere is one.
 
     ax2.text(offset, 0, r'$0$', horizontalalignment='center',
              verticalalignment='center', fontsize=14)
-    ax2.text(0, offset, r'$\pi/2$', horizontalalignment='center',
+    ax2.text(0, offset, r'$90$', horizontalalignment='center',
              verticalalignment='center', fontsize=14)
 
-    ax2.text(-offset, 0, r'$\pi$', horizontalalignment='center',
+    ax2.text(-offset, 0, r'$180$', horizontalalignment='center',
```

Comment:
It looks like this overlaps with the color wheel slightly, could you change this constant somehow so that it doesn't?

Reply:
This will need another PR. The issue is that the same offset is used for all four labels for the phase circle. For now, I put in a temporary way to do this by adding some trailing spaces after 180.